ui/packages/catalog-ui-search/src/main/webapp/component/query-item/query-item.less

```diff
@@ -66,21 +83,31 @@
   }
 
   .details-footer {
-    text-align: right;
+    padding-left: @minimumSpacing;
+    display: flex;
+    justify-content: space-between;
     line-height: @minimumButtonSize;
 
-    > button,
-    > .query-edit,
-    > .query-schedule,
-    > .query-settings,
-    > .query-actions {
+    button,
+    .query-edit,
+    .query-schedule,
+    .query-settings,
+    .query-actions,
+    .query-status {
       vertical-align: middle;
-      width: @minimumButtonSize;
```

Comment:
The reason I switched it to padding is so that the distance between each icon would measure the same distance. This gives the illusion that buttons are the same width. With the hardcoded button size the distance between each icon will vary depending on how much width the icon takes up out of the button width, and the buttons are perceived to be different widths even though they are not.  The padding being used ensures a minimum button width that has a decent clickable area, so it should not pose an issue.